edge/pkg/edged/edged.go

```diff
@@ -107,7 +107,7 @@ import (
 	edgecadvisor "github.com/kubeedge/kubeedge/edge/pkg/edged/cadvisor"
 	"github.com/kubeedge/kubeedge/edge/pkg/edged/clcm"
 	edgedconfig "github.com/kubeedge/kubeedge/edge/pkg/edged/config"
-	fakekube "github.com/kubeedge/kubeedge/edge/pkg/edged/fake"
+	fakekube "github.com/kubeedge/kubeedge/edge/pkg/edged/kubeclientbridge"
```

Comment:
suggest	
`clientbridge "github.com/kubeedge/kubeedge/edge/pkg/edged/kubeclientbridge"`

Reply:
> suggest `clientbridge "github.com/kubeedge/kubeedge/edge/pkg/edged/kubeclientbridge"`

Good suggestion！I've already modify it.